src/main/java/frc/robot/controls/Driver.java

```diff
@@ -37,17 +87,17 @@ public static double getSteer()
         return steerValue;
     }
 
-    public static boolean getThrottleButton(int button)
+    public boolean getThrottleButton(int button)
     {
         return throttle.getRawButton(button);
     }
 
-    public static boolean getSteerButton(int button)
+    public boolean getSteerButton(int button)
     {
         return steer.getRawButton(button);
     }
 
-    public static boolean getSteerButtonPressed(int button)
+    public boolean getSteerButtonPressed(int button)
     {
         return steer.getRawButtonPressed(button);
     }
```

Comment:
These should all be private.  If you need to get a certain button for a command, make a function that returns that button so you don't need to manually call these ugly functions.